tests/browser/environment.js

```diff
@@ -32,6 +32,9 @@ class PuppeteerEnvironment extends JestNodeEnvironment {
         this.global.page = this.page;
 
         this.assignTestGlobals();
+
+        // Close sentinel about:blank page created by Puppeteer but not used in tests.
+        (await this.browser.pages())[0].close();
```

Comment:
Good as is, but might be add a guard that the first page is `about:blank`

Reply:
I added this check, does it look better now?